### ISSUE TITLE
System.Xml.Xsl.XslTransformApi.Tests failed on netcoreapp5.0-Windows_NT-Debug-x64-Mono_release-Windows.7.Amd64.Open

### ISSUE THREAD

**Issue by am11** [dotnet/runtime#35177]
https://dev.azure.com/dnceng/public/_build/results?buildId=607333&view=ms.vss-test-web.build-test-results-tab&runId=19079934&paneView=attachments&resultId=177438

```
dotnet.exe.1604.dmp:
  https://helix.dot.net/api/2019-06-17/jobs/d6034611-b403-4c3d-898d-0942a2e7c646/workitems/System.Xml.Xsl.XslTransformApi.Tests/files/dotnet.exe.1604.dmp

console.ac718e3e.log:
  https://helix.dot.net/api/2019-06-17/jobs/d6034611-b403-4c3d-898d-0942a2e7c646/workitems/System.Xml.Xsl.XslTransformApi.Tests/files/console.ac718e3e.log
```

Failed in https://github.com/dotnet/runtime/pull/35173.

Going by the `History` menu, seems like first occurrence of this test+leg failure was on 02/22/2020 in https://dev.azure.com/dnceng/public/_build/results?buildId=533303&view=ms.vss-test-web.build-test-results-tab&runId=16931504&resultId=100203 (or maybe we have hit the history limit there), but it has failed multiple times since.

**Comment by buyaa-n** [comment]
@am11 none of the links in the description is working, is this failure happened again?

**Comment by am11** [comment]
GitHub has the record of failure: https://github.com/dotnet/runtime/runs/603516559 from April, looks like the build on DevOps side was deleted.

**Comment by buyaa-n** [comment]
> GitHub has the record of failure: https://github.com/dotnet/runtime/runs/603516559 from April, looks like the build on DevOps side was deleted.

Yeah seems its deleted and the Github record doesn't say anything about the failure, if the failure not happening anymore I would assume that was not caused by XML code or test.

**Comment by jkotas** [comment]
Failed in https://github.com/dotnet/runtime/pull/39538

**Comment by jaredpar** [comment]
Closing as there are no recent failures of this. 

https://runfo.azurewebsites.net/search/tests/?bq=definition%3Aruntime+started%3A%7E7&tq=System.Xml.Xsl.XslTransformApi.Tests
